src/style.css

```
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.bar {
+  font-weight: 900;
+  font-family: Avenir, sans-serif;

```

You can inherit font
```suggestion
  font-family: inherit;
```